code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -429,28 +450,28 @@ nobiliumsuppression = INFINITY
 	var/initial_plasma = cached_gases[/datum/gas/plasma][MOLES]
 	var/initial_hydrogen = cached_gases[/datum/gas/hydrogen][MOLES]
 	var/scale_factor = (air.volume)/(PI) //We scale it down by volume/Pi because for fusion conditions, moles roughly = 2*volume, but we want it to be based off something constant between reactions.
-	var/toroidal_size = (2*PI)+TORADIANS(arctan((air.volume-TOROID_VOLUME_BREAKEVEN)/TOROID_VOLUME_BREAKEVEN)) //The size of the phase space hypertorus
+	var/toroidal_size = (2 * PI) //The size of the phase space hypertorus
```

Comment:
yes, since the addition of tiered canisters fusion has been very wonky and instability was pretty much always endo (when starting so you needed way more heat than just 10k) 
i confronted this with froggy and it was good for them

Reply:
Aight, just wanted to check